### PR TITLE
Make some change(part 2)

### PR DESCRIPTION
**Pull request by MsHeavyFour**
This is a response to the previous https://github.com/raghavg123/chanim/pull/23 Is intended to add a description of Chemfig, so that users can quickly get started with ChemMobject and other classes. 
I hope this pull request can be merged. I can't wait.
PS: Both accounts are mine, one for the computer and one for the iPad.

### CODE REVIEW

**Comment by MsHeavyFour** [comment]
Although there seem to be many changes, most of them offset each other.

**Comment by MsHeavyFour** [comment]
事实上，我仍然希望删除README.md的印地语版本。原因如下：1。印地语不特别。就影响力而言，它不如西班牙语（西班牙语是南美洲的重要语言）；就用户数量而言，它不如中文好；就重要性而言，它甚至可能不如法语好。2。这可能会产生非常严重的影响。如果是中国人，我可能会想，“既然印地语有自己的README.md，为什么中国人不能拥有它？”中国人这么认为，法语、西班牙语和葡萄牙语也会这么认为。如果您同意在这些语言中添加README.md，它将占用网络空间；如果您不同意，他们将指责您“语言歧视”，并要求删除印地语README.md。这是一个两难境地。那么如何解决这个问题呢？一个很好的建议是创建一个创建新的分支，其中创建了印地语README.md。这不会干扰主分支，但也符合保留印地语README.md的条件。如果这个方案可以通过，我将删除印地语README.md，并创建一个包含印地语README.md的新分支。@kilacoda I hope this euphemism can be passed.

**Comment by MsHeavyFour** [comment]
I'm terribly sorry. Because of the problem of the translation interface, the last comment was displayed in Chinese. I still want to delete Hindi README.md.
Apologize again for the accident.

**Comment by kilacoda-old** [comment]
Your comment shows up entirely in Chinese to me right now 😅 but I think I can get what you're saying by looking at the original message. I've attached what I can see:

> If I were a Chinese, I might think, "since Hindi has its own README.md, why can't Chinese have it?" Chinese think so, French, Spanish and Portuguese will also think so. If you agree to add README.md in these languages, it will occupy network space; If you don't agree, they will accuse you of "language discrimination" and ask to delete Hindi README.md. This is a dilemma.
So how to solve this problem?
A good suggestion is to create a new branch in which the Hindi README.md is created. This will not interfere with the main branch, but also meet the conditions for retaining Hindi README.md.
If this scheme can pass, I will delete Hindi README.md, and create a new branch that includes Hindi README.md.

I'm not against other languages having their own READMEs, in fact I'm all for translation of content in open source projects into as many languages as possible. Hindi is the only language I know other than English so that's why I wrote one in it, but nobody's expressed interest for any other languages before so it's the only one there.

As for network space, I don't think a few extra README files are of concern for that. Ideally I'd not have READMEs included in the pip release, but if that's what's happening atm I'll look into it. Even big repos like @textualize/rich have many READMEs and they don't seem to have any problems with network space.

Creating a separate branch just for a README decreases visibility even for any future READMEs if added, so I'm not really on board with that idea, sorry.

If you feel it'll be helpful and are interested in writing a Chinese README, feel free to, I'd be happy to merge that!

**Comment by ghost** [comment]
I think your idea is very good. I will add Chinese README.md soon as possible.

**Comment by MsHeavyFour** [comment]
Take stock of the current progress:

1. Created the Chinese version of README;
2. Revised the README in Hindi and English to include the description of Chemfig (I dare not say how standard my Hindi is with the help of translation. If there is any mistake, please help correct it).
From above, I have every reason to believe that this pull request can be passed.
I now request that the pull request be reviewed immediately and approved.

File: README-हिन्दी.md [hunk 1]
```diff
@@ -41,15 +41,13 @@ manim -p -qm chem.py ChanimScene
...
-
+https://user-images.githubusercontent.com/65204531/124297601-dcafcf80-db78-11eb-936b-cdc913c91f25
 
 बधाई हों! आपने कैनिम के साथ अपने पहले एनिमेशन की रचना करी और उसे चलाया।
 
 कैनिम और उसके इस्तेमाल के बारे में और जानने के लिए आप सूत्र कोड और दस्तावेज़ (जल्द आने वाला है!) देख सकते हैं।
 
 ## क्षमताऐं
-अभी के लिए कैनिम केवल यौगिकों और प्रतिक्रियाओं, तथा कुछ और chemfig आदेशों (जैसे की समन्वय बॉन्ड आदि), परंतु और आना बाकी है! अगर आपके पास कोई सुझाव है, कृपया एक नया issue बनाये।
+अभी के लिए कैनिम केवल यौगिकों और प्रतिक्रियाओं, तथा कुछ और Chemfig आदेशों (Chemfig एक लेटेक्स पैकेज है जो रासायनिक रेणू को चित्र कर सकता है. विवरण दस्तावेज़ वेबसाइट है [यहाँ है]( http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf), परंतु और आना बाकी है! अगर आपके पास कोई सुझाव है, कृपया एक नया issue बनाये।
```
Comment (kilacoda) [inline README-हिन्दी.md:51]:
```suggestion
अभी के लिए _कैनिम_ केवल यौगिकों और प्रतिक्रियाओं, तथा कुछ और Chemfig आदेशों (Chemfig एक LaTeX पैकेज है जो रासायनिक रेणुओं चित्रण कर सकता है, और जिसका उपयोग _कैनिम_ अंदरूनी रूप से करता है)। इसका प्रलेखन [यहाँ]( http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf) उपलब्ध है) का चित्रण कर सकता है, परंतु और आना बाकी है! अगर आपके पास कोई सुझाव है, कृपया एक नया issue बनाये।
```

Corrected the Hindi a bit.

File: README.md [hunk 2]
```diff
@@ -48,7 +49,7 @@ Congrats! You've written and played your first animation with chanim (or "chanim
 Explore the code and docs (coming soon!) for more on how to use chanim.
 
 ## Abilities
-Currently chanim only supports drawing compounds and reactions along with a few chemfig commands (such as coordinate bonds and complexes etc.) but more is to come! If you have a suggestion, file an issue with a proper tag.
+Currently chanim only supports drawing compounds and reactions along with a few chemfig commands (Chemfig is a latex package that can draw chemical molecules such as coordination bonds. The description document website is [here](http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf)) but more is to come! If you have a suggestion, file an issue with a proper tag.
```
Comment (kilacoda) [inline README.md]:
```suggestion
Currently chanim only supports drawing compounds and reactions along with a few chemfig commands (Chemfig is a latex package that can draw chemical molecules and is used by chanim internally. The documentation is available [here](http://ctan.imsc.res.in/macros/generic/chemfig/chemfig-en.pdf)) but more is to come! If you have a suggestion, file an issue with a proper tag.
```
Fixed this a bit

File: README.md [hunk 3]
```diff
@@ -1,8 +1,9 @@
 # Chanim
 This is an extension to [Manim](https://www.github.com/ManimCommunity/manim) library (initially created by [3Blue1Brown](https://github.com/3b1b/manim)),
 for making videos regarding chemistry.
+> A Hindi version of this README is available [here](https://github.com/MsHeavyFour/chanim/blob/master/README-हिन्दी.md).
```
Comment (kilacoda) [inline README.md]:
```suggestion
> A Hindi version of this README is available [here](https://github.com/kilacoda/chanim/blob/master/README-हिन्दी.md).
```
link fix

File: README.md [hunk 4]
```diff
@@ -1,8 +1,9 @@
...
 
-> A Hindi version of this README is available [here](https://github.com/raghavg123/chanim/blob/master/README-%E0%A4%B9%E0%A4%BF%E0%A4%A8%E0%A5%8D%E0%A4%A6%E0%A5%80.md).
+> A Chinese version of this README is available [here](https://github.com/MsHeavyFour/chanim/blob/master/README-自述文件.md).
```
Comment (kilacoda) [inline README.md]:
```suggestion
> A Chinese version of this README is available [here](https://github.com/kilacoda/chanim/blob/master/README-自述文件.md).
```

File: README-自述文件.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+恭喜！你制作出了你的第一个Chanim视频（或者叫做Chanimation)!
+通过学习源码和将推出的文档以继续学习Chanim。
+## 功能
+
+目前Chanim仅支持绘制一些化学分子和Chemfig指令（Chemfig 是一个latex包，可以绘制配位键等化学分子。描述文档网站在[这里]（http://ctan.imsc.res.in /macros/generic/chemfig/chemfig-en.pdf)
+但我们会推出更多！如果您有建议或者问题，请用标签提交适当的issue。
+## 附记
+
+这个库目前仍不完善，可能有一些错误的地方。如果影响到了您的使用，欢迎提出自己的建议和修改。
```
Comment (kilacoda) [inline README-自述文件.md:48]: Seems mostly correct according to Google Translate, guess I'll have to take your word for it being accurate :)

**Review by kilacoda — CHANGES_REQUESTED** [review]

**Comment by kilacoda** [comment]
Sorry for the late review! Had finals and stuff to deal with. I made some changes (I think some things probably got mixed in translation such as "documentation"/"description document", but overall it seems pretty good. Thanks and good job!

File: README-自述文件.md (hunk 5 above)
Comment (MsHeavyFour) [reply README-自述文件.md:48]: Yes.

File: README-हिन्दी.md (hunk 1 above)
Comment (MsHeavyFour) [reply README-हिन्दी.md:51]: I must believe your Hindi.

**Review by MsHeavyFour — COMMENTED** [review]
I believe this changes is good.

**Comment by kilacoda** [comment]
I think everything is good to go, except that I don't believe it's necessary to add the chemfig manual to the repo, since we're already providing a link to it. Could you please remove this file from the PR? I'll merge it then.

**Comment by MsHeavyFour** [comment]
OK, I will do it.

**Comment by MsHeavyFour** [comment]
I deleted it.

**Review by kilacoda — APPROVED** [review]

**Pull request merged**